<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GooseParamSetup</class>
 <widget class="QDialog" name="GooseParamSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GOOSE分析配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>分析内容</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="5" column="2">
           <widget class="QCheckBox" name="checkBoxDsnameDiff">
            <property name="text">
             <string>数据集与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBoxConfrevDiffCfg">
            <property name="text">
             <string>版本号与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkBoxGocbrefDiffCfg">
            <property name="text">
             <string>gocbref与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBoxMacDiffCfg">
            <property name="text">
             <string>目的MAC与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBoxDaNumDiffCfg">
            <property name="text">
             <string>通道个数与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxAppidZero">
            <property name="text">
             <string>APPID为零</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBoxRepeat">
            <property name="text">
             <string>序号重复帧</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxDropPkt">
            <property name="text">
             <string>丢帧</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="checkBoxGoidDiffCfg">
            <property name="text">
             <string>goid与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxTremble">
            <property name="text">
             <string>心跳抖动</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkBoxTError">
            <property name="text">
             <string>发送机制错误</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QCheckBox" name="checkBoxDatypeDiffCfg">
            <property name="text">
             <string>数据类型与配置不一致</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBoxDelay">
            <property name="text">
             <string>GOOSE延迟</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxStNumReverse">
            <property name="text">
             <string>stNum逆转</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxTalZero">
            <property name="text">
             <string>TAL为零</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxDstMacErr">
            <property name="text">
             <string>目的MAC地址错误</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBoxSqNumJump">
            <property name="text">
             <string>sqNum跳变</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBoxStNumJump">
            <property name="text">
             <string>stNum跳变</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBoxRestart">
            <property name="text">
             <string>链路重启</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBoxNdscommTrue">
            <property name="text">
             <string>控制块需要配置</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxTest">
            <property name="text">
             <string>置检修标志</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxSqNumReverse">
            <property name="text">
             <string>sqNum逆转</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QCheckBox" name="checkBoxStNumChgFake">
            <property name="text">
             <string>虚假变位</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxSqNumUnreset">
            <property name="text">
             <string>GOOSE新事件，sqNum未复归</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QCheckBox" name="checkBoxStNumChgRight">
            <property name="text">
             <string>状态变位</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="checkBoxDouble">
            <property name="text">
             <string>重复帧</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="checkBoxDaentriesDiffDanum">
            <property name="text">
             <string>numDataEntries与实际数据个数不一致</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>启动门槛</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>心跳抖动</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditTrembleMinpercent"/>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEditDelay"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_2">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>报文延迟</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>丢帧</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditDroppkt"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_4">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>发送机制错误</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEditTError"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GooseParamSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GooseParamSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
